db/migrate/20230119024339_add_user_id_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] I think you should make use of author rather than user, because the foriegn key is AuthorId coming from the Author model which is an alias to User model.